src/header/NamespaceDropdown/NamespaceDropdown.tsx

```diff
@@ -1,18 +1,46 @@
 import { Menu, Icon } from 'fundamental-react';
-import { Link } from 'react-router-dom';
+import { useRecoilValue } from 'recoil';
+import { Link, useMatch } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
 
+import { namespacesState } from 'state/namespacesAtom';
 import { useUrl } from 'hooks/useUrl';
-import { NamespacesState } from 'state/namespacesAtom';
 
 import './NamespaceDropdown.scss';
 
+const Namespaces = () => {
+  const namespaces = useRecoilValue(namespacesState);
+  const { clusterUrl, namespace, namespaceUrl } = useUrl();
+
+  const { resourceType = '' } =
+    useMatch({
+      path: '/cluster/:cluster/namespaces/:namespace/:resourceType',
+      end: false,
+    })?.params ?? {};
+
+  const switchNamespace = (namespaceName: string) => {
```

Comment:
`switchNamespace` is a rather strange name for a function that simply returns a url instead of doing any switching. Although with the comment below this function might not be needed at all